geode-docs/tools_modules/gfsh/command-pages/import.html.md.erb

```diff
@@ -23,79 +23,103 @@ limitations under the License.
 
 You can import data into a region or import an existing cluster configuration into the cluster.
 
--   **[import data](../../../tools_modules/gfsh/command-pages/import.html#topic_jw2_2ld_2l)**
+-   **[import cluster-configuration](#topic_vnv_grz_ck)**
+
+    Import a cluster configuration.
+
+-   **[import data](#topic_jw2_2ld_2l)**
 
     Import user data from a file to a region.
 
--   **[import cluster-configuration](../../../tools_modules/gfsh/command-pages/import.html#topic_vnv_grz_ck)**
+## <a id="topic_vnv_grz_ck" class="no-quick-link"></a>import cluster-configuration
 
-    Import a cluster configuration.
+Imports a previously-exported cluster configuration from a zip file into a locator. This command is useful when spinning up a new cluster and when
+upgrading from an older version of <%=vars.product_name%> to a new one.
 
-## <a id="topic_jw2_2ld_2l" class="no-quick-link"></a>import data
+- In a given cluster, only one locator needs to perform the import. 
+That locator shares the imported configuration with all other connected locators in the distributed system.
+- Locators share the imported configuration with data members (servers) when they start, or if they have been recently started and no other
```

Comment:
Use of 'they' feels vague in this context.  Is the configuration shared as locators start or as servers start?